backend/Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN groupadd -g $USER_GID off && \
 
 COPY backend/requirements*.txt /code/
 COPY backend/editor /code/editor
+COPY backend/tests /code/tests
```

Comment:
I would say we don't need test in the image (no need for them in prod), but we should bind mount them in docker-compose.yml instead (so you never have to rebuild)

Reply:
I have pushed a commit to fix that.